polymer-element.js

```diff
@@ -28,3 +28,8 @@ export { html } from './lib/utils/html-tag.js';
  */
 export const PolymerElement = ElementMixin(HTMLElement);
 
+/**
+ * Current Polymer version in Semver notation.
+ * @type {String} Semver notation of the current version of Polymer.
+ */
+export const version = '3.0.5';
```

Comment:
Although this change makes introspecting the version from a running page possible, it is fairly cumbersome:
```js
import('./need-to-know-path-to-polymer/polymer-element.js').then(module => console.log(module.version));
```
And that only works if a build tool hasn't compiled modules into bundles or out altogether.

I think it would be pretty useful to have e.g. a static getter on the `ElementMixin` return the version (in addition to a plain module export), so that it's more straightforward to introspect the version being used on a running page.

e.g. from an element in the inspector
```js
console.log($0.constructor.polymerElementVersion);
```

or e.g. from the registry:
```js
console.log(customElements.get('my-app').polymerElementVersion);
```
So perhaps the version can be exported from `element-mixin.js` instead of here, and returned from a new `static get polymerElementVersion() { return version; }` getter on the `ElementMixin`?  It could also be re-exported from `polymer-element.js` for convenience.

Reply:
@kevinpschaaf Good idea! Updated.